### DISCUSSION TITLE
How to use Quspin for spin higher than spin 1/2

### DISCUSSION THREAD

**Discussion by magnon00** [QuSpin/QuSpin#450] [Q&A] [answered]
What are changes required in defining the  basis for spin higher than 1/2 
in following tutorial to study the exact diagonalisation of Spin Hamiltonians:

https://weinbe58.github.io/QuSpin/examples/example0.html#example0-label

I will be thankful to your help.

**Comment by JiahaoYao** [comment, marked as answer]
Hi @sachin00-github , thanks for asking! It might be useful for you to refer to the basis initialization params [page](https://weinbe58.github.io/QuSpin/generated/quspin.basis.spin_basis_1d.html#quspin.basis.spin_basis_1d.__init__). For example, you might find interesting in this. 
```markdown 
S: str, optional
	Size of local spin degrees of freedom. Can be any (half-)integer from: “1/2”,”1”,”3/2”,…,”9999/2”,”5000”.
```

**Reply by magnon00** [reply]
thanks @JiahaoYao

**Comment by mgbukov** [comment]
use the argument 'S=3/2' in the spin basis (check the online  documentation)
